samTemplates/amazon-connect-interactive-messages-example-lex-v2/packaged.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: amazon-connect-interactive-messages-example-lex-v2
+    Description: Creates AWS Lambda function to be used as an initialization and validation
+      code hook in Amazon Lex Bot (v2) for enabling interactive messages in Amazon
+      Connect Chat.
+    Author: Amazon Connect Messaging
+    SpdxLicenseId: MIT-0
+    LicenseUrl: s3://amazon-connect-interactive-message-blog-assets-lex-v2/7850c887b8e4c8f9654bdda1021d5d96
+    ReadmeUrl: s3://amazon-connect-interactive-message-blog-assets-lex-v2/6c34abba35f7d6cc18f2a7b03756d7e9
+    Labels:
+    - connect
+    - chat
+    - interactive-messaging
+    - contact-center
+    - lambda
+    - lex
+    - bot
+    HomePageUrl: https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/samTemplates/amazon-connect-interactive-messages-example-lex-v2
+    SemanticVersion: 2.0.0
+    SourceCodeUrl: https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/samTemplates/amazon-connect-interactive-messages-example-lex-v2
+Globals:
+  Function:
+    Timeout: 30
+Resources:
+  InteractiveMessagingLambdaFunction:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You must specify a valid value for Runtime (nodejs20.x). Valid values are [&#34;dotnet6&#34;, &#34;dotnetcore1.0&#34;, &#34;dotnetcore2.0&#34;, &#34;dotnetcore2.1&#34;, &#34;dotnetcore3.1&#34;, &#34;go1.x&#34;, &#34;java11&#34;, &#34;java17&#34;, &#34;java8&#34;, &#34;java8.al2&#34;, &#34;nodejs&#34;, &#34;nodejs10.x&#34;, &#34;nodejs12.x&#34;, &#34;nodejs14.x&#34;, &#34;nodejs16.x&#34;, &#34;nodejs18.x&#34;, &#34;nodejs4.3&#34;, &#34;nodejs4.3-edge&#34;, &#34;nodejs6.10&#34;, &#34;nodejs8.10&#34;, &#34;provided&#34;, &#34;provided.al2&#34;, &#34;python2.7&#34;, &#34;python3.10&#34;, &#34;python3.11&#34;, &#34;python3.6&#34;, &#34;python3.7&#34;, &#34;python3.8&#34;, &#34;python3.9&#34;, &#34;ruby2.5&#34;, &#34;ruby2.7&#34;, &#34;ruby3.2&#34;]